### PR TITLE
Add Komodo (KMD) coin

### PR DESCRIPTION
**Pull request by smdmitry**

### CODE REVIEW

**Comment by smdmitry** [comment]
I've made PR to BitGo: https://github.com/BitGo/bitgo-utxo-lib/pull/31
But no reaction from them so far.

**Comment by smdmitry** [comment]
BitGo won't merge any 3rd party contributions. Will you merge here?

File: src/coins.js [hunk 1]
```diff
@@ -32,7 +33,11 @@ coins.isLitecoin = function (network) {
 }
 
 coins.isZcash = function (network) {
-  return typeforce.value(coins.ZEC)(network.coin)
+    return typeforce.value(coins.ZEC)(network.coin) || this.isKomodo(network);
```
Comment (prusnak) [inline src/coins.js]: I don't like this. Do not modify the behaviour of this function.

**Comment by szymonlesisz** [comment]
@prusnak, already done in this commit:
https://github.com/trezor/trezor-utxo-lib/commit/c1e342fc9b1431a7bd1fcf635b0d375841713427

@smdmitry It would be nice if you provide some test fixtures for Komodo, same as "zcash" [here](https://github.com/trezor/trezor-utxo-lib/blob/trezor/test/fixtures/transaction.json#L884)
You can put them to ./test/fixtures/trezor.json file

**Comment by prusnak** [comment]
OP not reacting. Closing

**Pull request closed without merging**